Common/Core/RecoDecay.h

```diff
@@ -672,28 +667,10 @@ class RecoDecay
       return;
     }
     // If we are here, we have to follow the daughter tree.
-    //printf("getDaughters: ");
-    //for (int i = 0; i < stage; i++) // Indent to make the tree look nice.
-    //  printf(" ");
-    //printf("Stage %d: %d (PDG %d) -> %d-%d\n", stage, index, PDGParticle, indexDaughterFirst, indexDaughterLast);
```

Comment:
Do not remove the debugging prints. They are useful whenever a new decay is being added.

Reply:
Reverted